csrc/device_lower/analysis/tensor_memory.cpp

```diff
@@ -169,4 +181,90 @@ TensorMemoryInfo computeTMemInfo(Fusion* fusion) {
   return result;
 }
 
+std::pair<
+    std::unordered_map<TensorView*, TMemRegisterDataPath>,
+    std::unordered_map<TensorView*, TMemRegisterDataPath>>
+computeTMemLdStDataPath(Fusion* fusion) {
```

Comment:
I know this is just a placeholder for now, but I'm not entirely clear what this function is going to do.

Reply:
Added a comment to the code

Reply:
Thanks!